src/device-state/current-state.ts

```diff
@@ -177,7 +181,22 @@ const reportCurrentState = (): null => {
 		reportPending = true;
 		try {
 			const currentDeviceState = await deviceState.getStatus();
-			const info = await sysInfo.getSysInfoToReport();
+			const info = {
+				...((await config.get('disableHardwareMetrics'))
```

Comment:
You could avoid needing to re-fetch this every time using a mechanism like the target-state where it only updates when it's changed: https://github.com/balena-os/balena-supervisor/blob/v12.6.4/src/device-state/target-state.ts#L43-L58

Reply:
Good idea, done.